assets/src/scss/new-identity/style.scss

```diff
@@ -62,6 +62,7 @@
   --link--color: var(--grey-900);
   --link--visited--color: var(--grey-900);
   --link--hover--color: var(--grey-900);
+  --comment-reply-link--border-bottom: 2px solid var(--gp-green-500);
```

Comment:
Is this still needed since you added the `comment-reply-link` to this [selector](https://github.com/greenpeace/planet4-master-theme/pull/1959/files#diff-b418b2241107b99a37ed0291b0385d4ad8fe3762dd4d859be83bf83c070428d7R128)?

Reply:
I removed this variable and update [this line of code](https://github.com/greenpeace/planet4-master-theme/pull/1959/files#diff-b418b2241107b99a37ed0291b0385d4ad8fe3762dd4d859be83bf83c070428d7R129)